### PR TITLE
Create class and test

### PR DESCRIPTION
**Pull request by kevxo**
Creates class and test

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_literal_string: true
+
```
Comment (ShaunDaneJames) [inline lib/stat_tracker.rb]: We need to remove comments, Probably can be done at a later pull.

**Review by ShaunDaneJames — APPROVED** [review]

File: lib/stat_tracker.rb (hunk 1 above)
Comment (carson-jardine) [reply lib/stat_tracker.rb]: good catch!

**Comment by carson-jardine** [comment]
Looks great!

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# frozen_relative_string
```
Comment (Oxalisviolacea) [inline test/stat_tracker_test.rb]: Remove this comment too

**Review by Oxalisviolacea — COMMENTED** [review]
Remove comment

**Review by carson-jardine — APPROVED** [review]

**Review by carson-jardine — APPROVED** [review]

**Pull request merged**